web/models.py

```
@@ -1752,6 +1752,64 @@ def display_name(self):
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    creator = models.ForeignKey(User, on_delete=models.CASCADE)  # Creator field
+
+    def clean(self) -> None:
+        super().clean()
+
+        # Validate that link is provided for online meetups
+        if self.event_type == "online" and not self.link:
+            raise ValidationError("Online meetups require a meeting link")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)